benchmarks/flowertune-llm/README.md

```diff
@@ -0,0 +1,44 @@
+![](_static/flower_llm.jpg)
+
+# FlowerTune LLM Leaderboard
+
+This repository guides you through the process of federated LLM instruction tuning with a pre-trained [Mistral-7B](https://huggingface.co/mistralai/Mistral-7B-v0.3) models across various tasks.
+Please follow the instructions to run baseline tasks and create your own projects.
+
+
+## Structure
+Each LLM task within this directory is completely self-contained, with its source code neatly organized in its respective subdirectory.
+The task subdirectories contain the following structure:
+
+```bash
+flowertune-llm-leaderboard/<task-name>/
+                 ├── README.md
+                 ├── main.py         # <- Flower ClientApp/ServerApp build
+                 ├── client.py       # <- Flower client constructor
+                 ├── server.py       # <- Sever-related functions
+                 ├── model.py        # <- Model build
+                 ├── dataset.py      # <- Dataset and tokenizer build
+                 ├── pyproject.toml           # <- Environment dependencies
+                 ├── conf/config.yaml         # <- User configuration
+                 ├── static_conf/config.yaml  # <- Static configuration
+                 └── <evaluation>             # <- Evaluation metrics
```

Comment:
Now that we have the templates almost ready #3587 , what's the best way to present this structure? (and the instructions below). Also, are we replacing `"task"` with `"challenge"` or better to keep it ask `"task"`?